web/public/mgmt/config.yml

```diff
@@ -894,6 +894,19 @@ collections:
                       name: "sameContactCheckbox",
                       widget: "string",
                     }
+                  - { label: "Note Title", name: "noteTitle", widget: "string" }
+                  - { label: "Note Body", name: "noteBody", widget: "markdown" }
+                  - {
+                      label: "Checkbox Unchecked Screen Reader Announcement",
+                      name: "checkboxUnCheckedScreenReaderAnnouncement
+                        ",
+                      widget: "string",
+                    }
+                  - {
+                      label: "Checkbox Checked Screen Reader Announcement",
+                      name: "checkboxCheckedScreenReaderAnnouncement",
+                      widget: "string",
+                    }
```

Comment:
Did u check if these are searchable in the cms?

Reply:
Great callout, I double checked it and you were totally right, I had a syntax error in the search tool. Resolved now.